db/init/pg/create-table.sql

```diff
@@ -1604,11 +1604,11 @@ INSERT INTO "public"."shenyu_dict" VALUES ('1529402613195784281', 'saslMechanism
 -- ----------------------------
 DROP TABLE IF EXISTS "public"."user_role";
 CREATE TABLE "public"."user_role" (
-  "id" varchar(128) COLLATE "pg_catalog"."default" NOT NULL,
-  "user_id" varchar(128) COLLATE "pg_catalog"."default" NOT NULL,
-  "role_id" varchar(128) COLLATE "pg_catalog"."default" NOT NULL,
-  "date_created" timestamp(6) NOT NULL DEFAULT timezone('UTC-8'::text, (now())::timestamp(0) without time zone),
-  "date_updated" timestamp(6) NOT NULL DEFAULT timezone('UTC-8'::text, (now())::timestamp(0) without time zone)
+                                      "id" varchar(128) COLLATE "pg_catalog"."default" NOT NULL,
```

Comment:
not modify this. why format this sql?

Reply:
maybe occured by github reslove conflict, my local file is alright

Reply:
pls refused this pr

Reply:
Hi， 
1.you close this pr. 
2.pull remote master,
3. modify only the places you modified.
4. privoder new pull reuqest